i3lock.c

```diff
@@ -367,22 +367,26 @@ static void handle_key_press(xcb_key_press_event_t *event) {
     }
 
     switch (ksym) {
+        case XKB_KEY_j:
         case XKB_KEY_Return:
         case XKB_KEY_KP_Enter:
         case XKB_KEY_XF86ScreenSaver:
-            if (pam_state == STATE_PAM_WRONG)
-                return;
-
-            if (skip_without_validation()) {
-                clear_input();
+            if ((ksym == XKB_KEY_j && ctrl) ||
```

Comment:
I'd prefer if this was written as

````
if (ksym == XKB_KEY_j && !ctrl)
    break;

/* … */
```

That way we also avoid the indentation.